app/controllers/devise_token_auth/concerns/set_user_by_token.rb

```diff
@@ -125,6 +125,19 @@ def resource_class(m=nil)
     mapping.to
   end
 
+  def get_resource(uid_field='uid', uid=nil)
+    if DeviseTokenAuth.custom_scope
+      resource_class.send(DeviseTokenAuth.custom_scope, uid_field, uid).first
```

Comment:
yeah you're right, I thought I needed it for another case but I don't.  I'll remove it.